examples/standalone/gtest_setup/CMakeLists.txt

```diff
@@ -1,36 +1,33 @@
-cmake_minimum_required(VERSION 3.11.0 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.16 FATAL_ERROR)
 
 project(GTestSetup)
 
 # Find Gazebo
 find_package(gz-sim8 REQUIRED)
 set(GZ_SIM_VER ${gz-sim8_VERSION_MAJOR})
 
-# Fetch and configure GTest
-include(FetchContent)
-FetchContent_Declare(
-  googletest
-  URL https://github.com/google/googletest/archive/609281088cfefc76f9d0ce82e1ff6c30cc3591e5.zip
```

Comment:
I have no clue why google test still recommends fetchcontent. Google doesn't really use CMake internally. People should use the gtest supplied with their OS unless they have a good reason not to.

Reply:
Okay, but then we should probably update the README and to say that users should first install gtest.

Reply:
I'm afraid this breaks CI: 
* https://build.osrfoundation.org/job/gz_sim-ci-pr_any-homebrew-amd64/414/testReport/junit/(root)/Plugins_ExamplesBuild/Build_standalone_gtest_setup/
* https://build.osrfoundation.org/job/gz_sim-ci-pr_any-jammy-amd64/584/testReport/junit/(root)/Plugins_ExamplesBuild/Build_standalone_gtest_setup/

Probably because we don't install gtest in our CI machines. @j-rivero any thoughts on how to proceed? Is it better to revert this?

Reply:
Alternatively, we could add gtest as a build dependency to the `package.xml`, assuming dependencies are managed through rosdep.

Reply:
They are not handled via `package.xml`. That is a recent addition (#2337) and none of our CI currently uses it yet.

Reply:
> Probably because we don't install gtest in our CI machines

This is correct. We actually vendor a specific gtest version in each of our packages.